Sources/CacheAdvance/EncodableMessage.swift

```diff
@@ -40,7 +40,7 @@ struct EncodableMessage<T: Codable> {
     /// The encoded message, prefixed with the size of the message blob.
     func encodedData() throws -> Data {
         let messageData = try encoder.encode(message)
-        guard messageData.count < MessageSpan.max else {
+        guard messageData.count < Size.max else {
```

Comment:
Utilizing a generic here let me run a test where I passed in a message that would throw the error below without that message requiring multiple gigabytes of RAM.